orchestrator/api.toml

```diff
@@ -0,0 +1,38 @@
+[meta]
```

Comment:
Big fan of this file! Toml for defining endpoints is neat! (Is this parsed by the application?)

Reply:
It is parsed by the application!  But it actually is a feature of tide-disco, so I can't take credit for it.